tests/admin_filters/tests.py

```diff
@@ -737,7 +792,7 @@ def test_simplelistfilter(self):
         choices = list(filterspec.choices(changelist))
         self.assertEqual(choices[0]['display'], 'All')
         self.assertEqual(choices[0]['selected'], True)
-        self.assertEqual(choices[0]['query_string'], '?')
+        self.assertEqual(choices[0]['query_string'], '?publication-decade={0}'.format(sentinel))
```

Comment:
How about adding a `list_filter_sentinel` to the list filter links only if `list_filter_defaults` is used? Then changes like this won't be required, I think.